improver/nbhood/nbhood.py

```diff
@@ -521,33 +520,43 @@ def pad_and_unpad_cube(self, slice_2d: Cube, kernel: ndarray) -> Cube:
             1. Take the input slice_2d cube with the data, where 1 is an
                occurrence and 0 is an non-occurrence::
 
-                    [[1., 1., 1.,],
-                     [1., 0., 1.],
-                     [1., 1., 1.]]
+                    [
+                        [
+                            1.0,
+                            1.0,
+                            1.0,
+                        ],
+                        [1.0, 0.0, 1.0],
+                        [1.0, 1.0, 1.0],
+                    ]
```

Comment:
Only nbhood.py seems to have been upset by the switch to ruff. I think reverting these changes and adding `# fmt: off` and `# fmt: on` around these arrays will be sufficient